docker/install/ubuntu_install_python.sh

```diff
@@ -20,21 +20,17 @@ set -e
 set -u
 set -o pipefail
 
-# install python and pip, don't modify this, modify install_python_package.sh
-apt-get update
-apt-get install -y python-dev
-
-# python 3.6
+# python 3.7
 apt-get install -y software-properties-common
 
 add-apt-repository -y ppa:deadsnakes/ppa
 apt-get update
-apt-get install -y python-pip python-dev python3.6 python3.6-dev
+apt-get install -y python3.7 python3.7-dev
 
-rm -f /usr/bin/python3 && ln -s /usr/bin/python3.6 /usr/bin/python3
+rm -f /usr/bin/python3 && ln -s /usr/bin/python3.7 /usr/bin/python3
 
 # Install pip
-cd /tmp && wget -q https://bootstrap.pypa.io/get-pip.py && python3.6 get-pip.py
+cd /tmp && wget -q https://bootstrap.pypa.io/get-pip.py && python3.7 get-pip.py
 
```

Comment:
I suggest we change the Dockerfile to use `docker/install/ubuntu1804_install_python.sh` instead, rather than modifying this script, that might be used in other Dockerfiles.

Reply:
Sorry, I made little mess, so I reverted some code. I've already pushed changes. I reverted most of my code, and I have changed code, to use ubuntu1804_install_python

Reply:
No problem. Did you re-test the container build with the ubuntu1804_install_python.sh script and it works?